mitxpro/urls_test.py

```diff
@@ -1,12 +1,7 @@
 """Tests for URLs"""
 
 from unittest import TestCase
-from django.urls import reverse
 
 
 class URLTests(TestCase):
     """URL tests"""
```

Comment:
This whole file can be deleted. It's not doing anything for us

Reply:
Removed